registry/application/services/file_service/file_service.py

```diff
@@ -0,0 +1,62 @@
+import boto3
+
+from registry.application.services.file_service.constants import FileType
+from registry.exceptions import InvalidFileTypeException
+
+
+class FileService:
+
+    def __init__(self):
+        self.boto_client = boto3.client('s3')
+
+    def delete(self, file_details):
```

Comment:
There is no return from this method?

Reply:
yeah. need to add that.

Reply:
Whats the behaviour when the delete fails? Is an exception thrown?
Does bucket.objects.filter(Prefix=prefix).delete() return anything?

Reply:
[de68681](https://github.com/singnet/snet-marketplace-service/pull/1338/commits/de68681551ba51aaaa3b644ab8a6a025e57db9a9) added error handling for failed delete operations.
still returning success in function, presuming it will be partial error in deleting.

Reply:
[boto 1.10.9 docs](https://boto3.amazonaws.com/v1/documentation/api/1.10.9/reference/services/s3.html#S3.Bucket.objects)
delete() 
returns
- (list)
  - (dict) --
    - Deleted (list) -- 
      - (dict) --
        - Key (string) --
        - VersionId (string) --
        - DeleteMarker (boolean) --
        - DeleteMarkerVersionId (string) --
     - RequestCharged (string) -- 
       - If present, indicates that the requester was successfully charged for the request. 
     - Errors (list) -- 
       - (dict) --
         - Key (string) --
         - VersionId (string) --
         - Code (string) --
         - Message (string) --

```{
    'Deleted': [
        {
            'Key': 'string',
            'VersionId': 'string',
            'DeleteMarker': True|False,
            'DeleteMarkerVersionId': 'string'
        },
    ],
    'RequestCharged': 'requester',
    'Errors': [
        {
            'Key': 'string',
            'VersionId': 'string',
            'Code': 'string',
            'Message': 'string'
        },
    ]
}
```

Reply:
Other case of fail will be boto loses its connection. which will throw and error and it will be handled on lambda handler level